docs/tools/vdb_table/data/pinecone.json

```diff
@@ -68,38 +68,38 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/pinecone-io/canopy",
     "comment": ""
   },
   "embeddings_image": {
-    "support": "none",
-    "source_url": "",
+    "support": "",
+    "source_url": "https://github.com/pinecone-io/image-search-example",
     "comment": ""
   },
   "embeddings_structured": {
     "support": "",
-    "source_url": "",
+    "source_url": "https://github.com/pinecone-io/examples/blob/master/learn/search/question-answering/table-qa.ipynb",
     "comment": ""
   },
   "rag": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
```

Comment:
This column is to indicate an inbuilt functionality of the engine to run the entire RAG pipeline (text search query embedding --> search --> generation via LLM). This is separate from a langchain integration.

You can see an example in the Weaviate row for this.

Reply:
noted, will update